src/vulkan/vk-device.cpp

```
@@ -35,6 +36,95 @@ static constexpr VkSubgroupFeatureFlags kWaveOpsSubgroupFeatureMask =
+    if (getCpuTimestampDomain() == CpuTimestampDomain::QueryPerformanceCounter)
+    {
+        qpcFrequency = getCpuTimestampFrequency();
+    }
+    candidates.push_back(
+        {VK_TIME_DOMAIN_QUERY_PERFORMANCE_COUNTER_KHR, CpuTimestampDomain::QueryPerformanceCounter, qpcFrequency}
+    );
+#endif
+#if SLANG_LINUX_FAMILY
+    candidates.push_back(
+        {VK_TIME_DOMAIN_CLOCK_MONOTONIC_RAW_KHR, CpuTimestampDomain::ClockMonotonicRaw, kNanosecondsPerSecond}
+    );
```

Cppcheck (2.20.0)

[style] The function '_Move' is never used.

(unusedFunction)